_data/devops-checklist.yml

```diff
@@ -0,0 +1,690 @@
+- category: Server-side
+  items:
+    - title: Build AMIs
+      description: |
+        If you want to run your apps directly on EC2 Instances, you should package them as
+        <a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html" target="_blank">Amazon Machine Images (AMIs)</a>
+        using a tool such as <a href="https://www.packer.io/" target="_blank">Packer</a>. Although we recommend Docker
+        for all stateful apps (see below), we recommend directly using AMIs and EC2 Instances for all stateful apps,
+        such as any data store (MySQL, MongoDB, Kafka), and app that writes to its local disk (e.g., WordPress, Jenkins).
+
+    - title: Deploy AMIs using Auto Scaling Groups
+      description: |
+        The best way to deploy an AMI is typically to run it as an
+        <a href="https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html" target="_blank">Auto Scaling Group</a>.
+        This will allow you to spin up multiple EC2 Instances that run your AMI, scale the number of Instances up and
+        down in response to load, and automatically replace failed Instances.
+
+    - title: Build Docker images
+      description: |
+        If want to run your apps as containers, you should package your apps as
+        <a href="http://docker.com/" target="_blank">Docker</a> images and push those images to Amazon's
+        <a href="https://aws.amazon.com/ecr/" target="_blank">Elastic Container Registry (ECR)</a>. We recommend
+        Docker for all stateless apps and for local development (along with
+        <a href="https://docs.docker.com/compose/" target="_blank">Docker Compose</a>).
+
+    - title: Deploy Docker images using ECS, EKS, or Fargate
+      description: |
+        You have several options for running Docker containers in AWS. One is to use the
+        <a href="https://aws.amazon.com/ecs/" target="_blank">Elastic Container Service (ECS)</a>, where you run a
+        cluster of EC2 Instances, and Amazon takes care of scheduling containers across them. Another is
+        <a href="https://aws.amazon.com/eks/" target="_blank">Elasic Kubernetes Service (EKS)</a>, which is a Amazon's
+        managed <a href="https://kubernetes.io/" target="_blank">Kubernetes</a> (note, EKS is still in preview mode
+        as of May, 2018). A third option is <a href="https://aws.amazon.com/fargate/" target="_blank">AWS Fargate</a>,
+        a service where AWS manages and scales the underlying Instances for you and you just hand it Docker containers
+        to run.
+
+    - title: Deploy serverless apps using Lambda and API Gateway
+      description: |
+        If want to build serverless apps, you should package them as
```

Comment:
If _you_ want to build...

Reply:
Fixed! https://github.com/gruntwork-io/gruntwork-io.github.io/pull/45/commits/ac969266fa45512affe5434f70111b488f8c4eb8